--- node_modules/ganache-cli/build/cli.node.js	2017-12-13 16:40:55.281461784 +0300
+++ cli.node.js	2017-12-13 16:43:16.308209070 +0300
@@ -90906,7 +90906,8 @@
     if (err) return callback(err);
 
     self.snapshots.push({
-      blockNumber: blockNumber
+      blockNumber: blockNumber,
+      timeAdjustment: self.blockchain.timeAdjustment
     });
 
     self.logger.log("Saved snapshot #" + self.snapshots.length);
@@ -90920,6 +90921,7 @@
 
   // Convert from hex.
   snapshot_id = utils.bufferToInt(snapshot_id);
+  var timeAdjustment = this.snapshots[snapshot_id].timeAdjustment;
 
   this.logger.log("Reverting to snapshot #" + snapshot_id);
 
@@ -90956,6 +90958,8 @@
 
     // Pending transactions are removed when you revert.
     self.blockchain.clearPendingTransactions();
+    // The time adjustment is restored to its prior state
+    self.blockchain.timeAdjustment = timeAdjustment;
 
     callback(null, true);
   });
@@ -106060,4 +106064,4 @@
 
 /***/ })
 /******/ ]);
-//# sourceMappingURL=cli.node.js.map
\ В конце файла нет новой строки
+//# sourceMappingURL=cli.node.js.map
